registration/templates/registration/failed_email.txt

```diff
@@ -0,0 +1,9 @@
+{% autoescape off %}
+Hi!
+
+The instance for {{ application.user.name }} failed to provision.
```

Comment:
Is this email template even used? I don't see where it is. 

If not, remove it. If it is to be used, this language won't really be helpful to end-users if it ever goes to them.

Reply:
Yeah, it was just a test that slip into that commit.